packages/jest-cli/src/reporters/__tests__/__snapshots__/utils-test.js.snap

```diff
@@ -8,9 +10,9 @@ exports[`trimAndFormatPath() trims dirname 1`] = `"[2m...234567890/[22m[1m123
 
 exports[`trimAndFormatPath() trims dirname and basename 1`] = `"[1m...1234.js[22m"`;
 
-exports[`wrapAnsiString() returns the string unaltered if given a terminal width of zero 1`] = `"This string shouldn\'t cause you any trouble"`;
+exports[`wrapAnsiString() returns the string unaltered if given a terminal width of zero 1`] = `"This string shouldn't cause you any trouble"`;
```

Comment:
Wait, actually this is correct now. I don't understand why it wasn't updated before. cc @vjeux.